server/src/rooms/game/state/index.ts

```diff
@@ -1226,7 +1234,7 @@ export class GameState extends Schema implements GameData {
       case 'sent-trade-request':
         message = `The ${fromRole} sent a trade request to the ${toRole}.`;
         category = MarsLogCategory.sentTrade;
-        this.log(message, category, performedBy);
+        this.log(round, message, category, performedBy);
```

Comment:
all of these calls to log won't need to pass in the round anymore since it's implicit in the GameState